pkg/compose/plugins.go

```diff
@@ -70,6 +74,8 @@ func (s *composeService) runPlugin(ctx context.Context, project *types.Project,
 		return err
 	}
 
+	mux.Lock()
+	defer mux.Unlock()
```

Comment:
We don't want providers to run sequentially, only the project mutation must be guarded by a mutex